docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+# Creating the vSphere Windows VM Golden Image
+
+This guide describes the thought process of creating a Windows virtual machine base image (the Golden Image) in vSphere.
+
+
+## 1. Select a compatible Windows Server version
+
+Currently the Windows Machine Config Operator (WMCO) stable version [2.0.0](REF?) only supports Windows Server version 1909 (build 18363.\*) Semi-Annual Channel, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), required to get the operating system feature that allows setting the [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere)
+
+*Please note that the above patch is not available for Windows Server version 2019.*
+
+> Comming soon: Support for Windows Server versions 2004 (build 19041.\*) and 20H2 (build 19042.\*) Semi-Annual Channels.
+
+
+## 2. Create the virtual machine
+
+To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
+See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html)
+
+
+### Setup VMware Tools
+
+Install VMware Tools version 11.0.6 or greater. See the [VMware Tools documentation](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-D8892B15-73A5-4FCE-AB7D-56C2C90BD951.html) for more information.
+
+Enable the VMware Tools configuration file [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) within the Windows virtual machine.
+
+For example in Windows Server version 1909 the configuration file is located at
+
+```powershell
+    C:\ProgramData\VMware\VMware Tools\tools.conf
+```
+
+In case the configuration file does not exist, VMware Tools installs an example configuration file in the same 
+directory as the location above or you can download it from the [open-vm-tools](https://raw.githubusercontent.com/vmware/open-vm-tools/master/open-vm-tools/tools.conf) repository.
+
+Ensure the virtual machine has a valid IP address in vCenter.
```

Comment:
Please call out how you do that here.

Reply:
That's definitely a good addition. However, I feel it should go in a separated/following PR.

Reply:
Why? You are adding this information here without specifying how you did it.